### ISSUE TITLE
Errors using Start= setting

### ISSUE THREAD

**Issue by LadiesMan217** [RPGHacker/asar#324]
!rb as2 -s
org $00f606
autoclean jsl derp
freecode start=$018000
derp:
nop
randombot999
APP
 — Today at 6:13 PM
Errors occurred while assembling.
<input>:3: error: (Eautoclean_label_at_freespace_end): Don't autoclean a label at the end of a freespace block, you'll remove some stuff you're not supposed to remove.
    in block: [autoclean jsl derp]

**Comment by trillllian** [comment]
oops, turns out asar doesn't like it when rats tags end up in banks $00-$0f.

should rats tags be allowed there?? in an smw context i think it's quite likely to be a mistake, but might be useful for homebrew? except who uses rats tags in a homebrew???

i guess i could turn it into a warning.

**Comment by p4plus2** [comment]
I mean, i don't see a reason to ban them.  There are 256kb snes games too.

For homebrew?  Probably not that useful, but, if we added a homebrew directive maybe?  Basically a build from source that clears the rom, then allocate using the free space commands.  I don't see that much use though.  Almost everything this solves could be done in other ways

**Comment by trillllian** [comment]
yeah after thinking about it a bit more i also decided to allow it.

re: "a homebrew command": i think we should probably just have a commandline option for "delete the output file before starting". or maybe implement something to allow specifying input and output rom names separately.

**Comment by p4plus2** [comment]
Cli flag is also fine imo.  We could stand to add quite a few (max errors for example)
